packages/core/src/components/carousel/carousel.tsx

```diff
@@ -64,15 +73,34 @@ export class AtomCarousel {
     this.swiperEl.swiper?.on('navigationNext', () => {
       this.atomClickNext.emit()
     })
+
+    let params: { pagination: PaginationOptions } = this.pagination && {
+      pagination: {
+        type:
+          this.paginationType === 'thumbnails' ? 'custom' : this.paginationType,
```

Comment:
(suggestion) `custom` instead `thumbnails` not needs this verification

Reply:
like i said in the comment below, i believe that this type is needeed so we dont lose the type custom